### PR TITLE
Transpose the VM rays so they can be drawn in NUsight fast

### PR DESCRIPTION
**Pull request by TrentHouliston**
NUsight needs the rays as columns so it can draw them, so we need to transpose them over here

### CODE REVIEW

File: roles/robocup.role [hunk 1]
```diff
@@ -3,51 +3,50 @@ NUCLEAR_ROLE(
     extension::FileWatcher
     support::SignalCatcher
     support::logging::ConsoleLogHandler
+
```
Comment (Bidski) [inline roles/robocup.role]: Why is this file changed?

Comment (TrentHouliston) [reply roles/robocup.role]: Shouldn't have been
